### PR TITLE
Updated code quality removed print statement spaces in singleinstance.py

### PR DESCRIPTION
**Pull request by kdcis**
Removed print statement spaces

### CODE REVIEW

File: src/singleinstance.py [hunk 1]
```diff
@@ -75,7 +75,7 @@ def lock(self):
                     fcntl.lockf(self.fp, fcntl.LOCK_EX | fcntl.LOCK_NB)
                 self.lockPid = os.getpid()
             except IOError:
-                print ('Another instance of this application is already running')
+                print('Another instance of this application is already running')
```
Comment (g1itch) [inline src/singleinstance.py:78]: `sys.exit('Another instance of this application is already running')`

**Comment by g1itch** [comment]
What about #1373?

File: src/singleinstance.py (hunk 1 above)
Comment (g1itch) [reply src/singleinstance.py:78]: And the same above. `print(e.errno)` is also redundant as for me.

Comment (kdcis) [reply src/singleinstance.py]: Updated

File: src/singleinstance.py [hunk 2]
```diff
@@ -55,12 +55,7 @@ def lock(self):
                 )
             except OSError as e:
                 if e.errno == 13:
-                    print(
-                        'Another instance of this application'
-                        ' is already running'
-                    )
-                    sys.exit(-1)
-                print(e.errno)
+                    sys.exit('Another instance of this application is already running')
```
Comment (g1itch) [inline src/singleinstance.py:58]: PEP8 violation here: line too long

File: src/singleinstance.py [hunk 3]
```diff
@@ -75,8 +70,7 @@ def lock(self):
                     fcntl.lockf(self.fp, fcntl.LOCK_EX | fcntl.LOCK_NB)
                 self.lockPid = os.getpid()
             except IOError:
-                print ('Another instance of this application is already running')
-                sys.exit(-1)
+                sys.exit('Another instance of this application is already running')
```
Comment (g1itch) [inline src/singleinstance.py:73]: line too long

File: src/singleinstance.py (hunk 2 above)
Comment (kdcis) [reply src/singleinstance.py]: Updated

File: src/singleinstance.py (hunk 3 above)
Comment (kdcis) [reply src/singleinstance.py]: Updated

File: src/singleinstance.py [hunk 4]
```diff
@@ -99,7 +97,7 @@ def cleanup(self):
                 pass
 
             return
-        print ("Cleaning up lockfile")
+        print("Cleaning up lockfile")
```
Comment (PeterSurda) [inline src/singleinstance.py:100]: just remove

**Review by PeterSurda — CHANGES_REQUESTED** [review]

File: src/singleinstance.py (hunk 4 above)
Comment (kdcis) [reply src/singleinstance.py]: updated

**Review by PeterSurda — APPROVED** [review]

**Review by PeterSurda — APPROVED** [review]

**Pull request merged**